spec/models/user_spec.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
```

Comment:
add spec for association and enum values.

Reply:
@kth-wtag Done, apu.